docs/running-on-kubernetes.md

```diff
@@ -990,6 +990,34 @@ See the [configuration page](configuration.html) for information on Spark config
   Specify whether executor pods should be deleted in case of failure or normal termination.
   </td>
 </tr>
+<tr>
+  <td><code>spark.kubernetes.submission.connectionTimeout</code></td>
+  <td>10000</td>
```

Comment:
The default seems to be here. (Maybe, does it come from `okhttp` originally) ?
- https://github.com/fabric8io/kubernetes-client/blob/master/kubernetes-client/src/main/java/io/fabric8/kubernetes/client/Config.java#L158-L159
```
  private int connectionTimeout = 10 * 1000;
  private int requestTimeout = 10 * 1000;
```